src/sql/src/plan/statement.rs

```diff
@@ -341,7 +341,7 @@ pub fn plan(
         Statement::ReassignOwned(stmt) => acl::plan_reassign_owned(scx, stmt),
 
         // DML statements.
-        Statement::Copy(stmt) => dml::plan_copy(scx, stmt),
+        Statement::Copy(stmt) => dml::plan_copy(scx, stmt, params),
```

Comment:
This is at odds with postgres and some drivers (tokio-postgres) don't support COPY params. I'm still fine with it, but it's worth pointing out.

Reply:
Probably worth not rocking the boat here. I just wanted to share some code with `plan_select_inner`, not change behavior. Thanks for spotting this. Pushed up a revision that passes in an empty params instead.